lib/foreman_maintain/cli/service_command.rb

```diff
@@ -0,0 +1,125 @@
+module ForemanMaintain
+  module Cli
+    class ServiceCommand < Base
+      option '--exclude', "EXCLUDE", 'A comma-separated list of services to skip'
+      option '--only', "ONLY", 'A comma-separated list of services to include'
+
+      subcommand 'start', 'Start applicable services' do
+        # TODO: DRY up labels
+        label_option
+        tags_option
+        interactive_option
+
+        def execute
+          run_scenario(start_scenario)
+          exit runner.exit_code
+        end
+      end
+
+      subcommand 'stop', 'Stop applicable services' do
+        label_option
+        tags_option
+        interactive_option
+
+        def execute
+         run_scenario(stop_scenario)
+          exit runner.exit_code
+        end
+      end
+
+      subcommand 'restart', 'Restart applicable services' do
+        label_option
+        tags_option
+        interactive_option
+
+        def execute
+          run_scenario([stop_scenario, start_scenario])
```

Comment:
It maybe useful to support the restart action even in the procedure or have separate procedure for it. Restart of the service is quite common task that may be used on other scenarios

Reply:
@mbacovsky I'll play around with it, but I do worry about repeating code with another procedure. This may be a good topic for us to discuss next week :smiley: